data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+#   Settings are meant to be controlled via OpenQA variables and managed by test:
+#   tests/sles4sap/publiccloud/qesap_ansible.pm
+
+provider: 'azure'
+apiver: 2
+terraform:
+  provider: 'azure'
+  variables:
+    # GENERAL VARIABLES #
+    az_region: "%PUBLIC_CLOUD_REGION%"
+    admin_user: "cloudadmin"
```

Comment:
Coding style consistent: using 'cloudadmin' instead of "cloudadmin"

Reply:
I kind of agree, but we should focus on more important things.

Reply:
Thanks that is no problem. I just did a mass replace and will be fixed in next push.

Reply:
And fixed.